webui/react/src/components/NavigationSideBar.tsx

```diff
@@ -317,6 +313,22 @@ const NavigationSideBar: React.FC = () => {
                           </li>
                         </WorkspaceActionDropdown>
                       ))}
+                    {canCreateWorkspace ? (
+                      <li>
+                        <NavigationItem
+                          icon="add-small"
+                          iconSize="tiny"
+                          label="New Workspace"
+                          labelRender={
+                            <Typography.Paragraph ellipsis={{ rows: 1, tooltip: true }}>
+                              New Workspace
+                            </Typography.Paragraph>
+                          }
+                          tooltip={settings.navbarCollapsed}
```

Comment:
this could be a trivial thing, but tooltip is be displayed in collapsed nav bar
this is the current behavior

Reply:
should it be changed? I was trying to copy other uses of tooltip

Reply:
no need to change i guess?
i just noticed it